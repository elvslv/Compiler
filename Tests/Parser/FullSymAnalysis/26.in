var i, j, n, l, k: integer;
r: real;
const m = 18;
begin
	for i := 1 to m do
		l := l + k;
end.