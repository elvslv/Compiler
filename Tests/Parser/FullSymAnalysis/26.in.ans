var I : INTEGER;
var J : INTEGER;
var N : INTEGER;
var L : INTEGER;
var K : INTEGER;
var R : REAL;
const M = 18;

begin                
|                    
|___for              
|   |                
|   |___:=           
|   |   |            
|   |   |___I        
|   |   |            
|   |   |___1        
|   |                
|   to               
|   |                
|   |___M            
|   |                
|   do               
|   |                
|   |___begin        
|       |            
|       |___:=       
|       |   |        
|       |   |___L    
|       |   |        
|       |   |___+    
|       |       |    
|       |       |___L
|       |       |    
|       |       |___K
|       |            
|       end          
|                    
end                  
