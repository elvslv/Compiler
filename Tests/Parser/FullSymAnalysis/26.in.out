var I : INTEGER;
var J : INTEGER;
var K : INTEGER;
var L : INTEGER;
const M = 18;
var N : INTEGER;
var R : REAL;

begin                
|                    
|___for              
|   |                
|   |___:=           
|   |   |            
|   |   |___I        
|   |   |            
|   |   |___1        
|   |                
|   to               
|   |                
|   |___M            
|   |                
|   do               
|   |                
|   |___begin        
|       |            
|       |___:=       
|       |   |        
|       |   |___L    
|       |   |        
|       |   |___+    
|       |       |    
|       |       |___L
|       |       |    
|       |       |___K
|       |            
|       end          
|                    
end                  
